src/ILAMB/ConfBasin.py

```diff
@@ -8,6 +8,7 @@
 import numpy as np
 import os
 from cf_units import Unit
+import rasterio
```

Comment:
This import isn't needed and causes test failures. This is because when all confrontations get imported in the Scoreboard, these imports are run. Originally we had the `try` and `except` block(s) inside the calls themselves. We could put this block up here at the head of the file and then remove them from the calls below.